src/ui/React/DialogBox.tsx

```diff
@@ -3,7 +3,7 @@ import { AlertEvents } from "./AlertManager";
 import React from "react";
 import { Typography } from "@mui/material";
 
-export function dialogBoxCreate(txt: string | JSX.Element, html = false): void {
+export function dialogBoxCreate(txt: string | JSX.Element, html = false, cancellable = true): void {
```

Comment:
This is starting to get a stack of boolean args, which are easy to confuse for each other. It's probably OK for now, but if we ever feel the need to add something else here, this should become an options struct.